modules/Virtualizer_module/CMakeLists.txt

```diff
@@ -17,14 +24,8 @@ add_install_rpath_support(BIN_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BIND
   DEPENDS ENABLE_RPATH
   USE_LINK_PATH)
 
-# find virtualizer SDK
-file(GLOB CybSDK_SOURCE_H "${CMAKE_CURRENT_SOURCE_DIR}/CybSDK/include/*")
-file(GLOB CybSDK_SOURCE_C "${CMAKE_CURRENT_SOURCE_DIR}/CybSDK/src/*")
-
 add_definitions(-DUNICODE)

```

Comment:
Probably this is not needed anymore. If it is needed, you can use [``target_compile_definitions``](https://cmake.org/cmake/help/latest/command/target_compile_definitions.html)